### PR TITLE
tools/doclets-tags-next-docs

### PR DESCRIPTION
**Pull request by KacperMadej**
Added info about doclets tags using next and linted.

### CODE REVIEW

**Review by ddragula — APPROVED** [review]
👍

File: ts/DOCLETS.md
Comment (AndrzejBuleczka) [inline ts/DOCLETS.md:1]: Thanks for adding this one!

**Review by AndrzejBuleczka — APPROVED** [review]
👍

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**